lnrpc/lightning.proto

```diff
@@ -2163,13 +2163,36 @@ message CloseChannelRequest {
     // NOTE: This field is only respected if we're the initiator of the channel.
     uint64 max_fee_per_vbyte = 7;
 
-    // If true, then the rpc call will not block while it awaits a closing txid
-    // to be broadcasted to the mempool. To obtain the closing tx one has to
-    // listen to the stream for the particular updates. Moreover if a coop close
-    // is specified and this flag is set to true the coop closing flow will be
-    // initiated even if HTLCs are active on the channel. The channel will wait
-    // until all HTLCs are resolved and then start the coop closing process. The
-    // channel will be disabled in the meantime and will disallow any new HTLCs.
+    // If `no_wait=true` (`wait=false`) when a coop close is attempted, then the
+    // rpc call will not initially block while it awaits a closing txid to be
+    // broadcasted to the mempool. Instead, a coop close will be initiated even
+    // if there are HTLCs active in flight on the channel and a `close_instant`
+    // will be immediately returned which contains `num_pending_htlcs` if there
+    // are in flight HTLCs that need to be taken out of flight before a coop
+    // close can be initiated. After the `close_instant` is received, then the
+    // channel will be disabled to disallow any new HTLCs and the rpc will block
+    // while it waits for the exsting in flight HTLCs to be resolved and then
+    // start the coop closing process. To obtain the closing TXID one has to
+    // listen to the stream for a `close_pending` update. If the client
+    // disconnects the stream before this is received, the rpc will continue to
+    // wait for the in flight HTLCs to be resolved and then start the coop
+    // closing process. Note: `lncli closechannel` always sets `no_wait=true`
+    // and its `--block` option controls if `lncli` should wait for a
+    // `close_pending` update and a `chan_close` update before returning or if
```

Comment:
should wait for the chan_close update (we still on the lncli side wait for the close_pending, which signals when the tx is broadcasted to the mempool)